extension/src/popup/components/account/AccountHeader/styles.scss

```diff
@@ -53,32 +54,10 @@ $dropdown-animation: 0.3s ease-out;
     top: 0;
     left: 0;
     z-index: calc(var(--z-index-tooltip) + 1);
-    transition: max-height $dropdown-animation;
+    transition: max-height var(--dropdown-animation);
 
     // default max-height on page load
     max-height: 0;
-
-    li::before {
```

Comment:
this `li::before` is still needed, the account header shows the dashed bullets when it's removed. Should we remove to global?
![Screen Shot 2022-06-15 at 9 33 33 AM](https://user-images.githubusercontent.com/30449853/173887085-f4372200-2d8a-4003-94b9-7440c41c3ec4.png)

Reply:
I just restored this style here rather than making it global on the off-chance this breaks something else